APIManager.js

```diff
@@ -0,0 +1,80 @@
+class APIManager {
+  constructor() {
+    this.data = {
+      userData: '',
+      baconData: '',
+      kenyaData: '',
+      pokemonData: '',
+    };
+  }
+
+  randomUsers() {
+    const getData = (response) => {
+      this.data.userData = response.results;
+    };
```

Comment:
you don't need this function, you could just "put" it inside the success function. -->> that goes for all the "getData" functions you declared later on in this file.

Besides that - you are saving ALL the information from the API (for this API and later on other APIs)  but that's not good practice - the data variable should hold only the information which you're gonna use in your project